CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 ### Fixed
-- Styling on Bolt checkout modal on iPhones >= 10
+- missing display id in request to get bolt checkout token
```

Comment:
this should be the title of the PR and not our ticket number. i changed it for you